libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -1497,9 +1492,9 @@ TEST( Full_PKCS11_EC, AFQP_Sign )
     CK_OBJECT_HANDLE xPublicKeyHandle;
     CK_OBJECT_HANDLE xCertificateHandle;
     /* Note that mbedTLS does not permit a signature on all 0's. */
```

Comment:
I don't think that's accurate. Rather, the issue is in trying to do the DSA computation with a zero exponent.

Reply:
thanks for pointing that out- fixed the comment here and other places.